src/BullOak.Repositories.EventStore.Test.Integration/Contexts/InProcEventStoreIntegrationContext.cs

```diff
@@ -81,13 +97,13 @@ public async Task AppendEventsToCurrentStream(Guid id, MyEvent[] events)
 
         public ResolvedEvent[] ReadEventsFromStreamRaw(Guid id)
         {
-            var connection = GetConnection();
+            var conn = GetConnection();
             var result = new List<ResolvedEvent>();
             StreamEventsSlice currentSlice;
             long nextSliceStart = StreamPosition.Start;
             do
             {
-                currentSlice = connection.ReadStreamEventsForwardAsync(id.ToString(), nextSliceStart, 100, false).Result;
+                currentSlice = conn.ReadStreamEventsForwardAsync(id.ToString(), nextSliceStart, 100, false).Result;
```

Comment:
This should be awaited. Is this a problem? Result was OK before because of the in-mem server, but now we should fix this.